packages/connect/src/data/defaultFeeLevels.ts

```diff
@@ -102,8 +102,8 @@ const SOLANA_FEE_INFO: FeeInfoWithLevels = {
             blocks: -1,
         },
     ],
-    minFee: -1, // unused
-    maxFee: -1, // unused
+    minFee: 25000,
+    maxFee: 1000000000,
```

Comment:
how come that suite didn't need this? 🤔

Reply:
There is probably no validation on this. And the fees are determined automatically by fetching the on chain data

Reply:
ok, good with me 👍